pkg/controller/crd_watcher.go

```diff
@@ -36,6 +37,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	EXTENSION_ROUTE_FALL_BACK = "envoy.router.cluster_specifier_plugin.cluster_fallback"
```

Comment:
Maybe this should be moved to `traffic_router_transform.go`?

Also please polish the naming according to the https://go.dev/doc/effective_go#names